.buildkite/release-pipeline.yml

```diff
@@ -68,6 +93,20 @@ steps:
           - "mkdir -p .artifacts"
           - buildkite-agent artifact download '.artifacts/*.tar.gz*' .artifacts/ --step build_docker_image_arm64
           - ".buildkite/publish/test-docker.sh"
+
+  - group: "Release repo cleanup"
```

Comment:
Do you mean `Release branch cleanup` here? Or just `Release cleanup` (as we have `Release setup`)?

Reply:
`Release cleanup` would be more consistent. The intent is to make sure that the repo/branch isn't left in a weird state, even if the tests or packaging fail. I don't want to leave the VERSION file as `8.16.0-build2024111012` on the HEAD of the `8.16` branch 😅